### ISSUE TITLE
[brats22/nnunet/pytorch] prediction problem with the post-processing in brats 22

### ISSUE THREAD

**Issue by abbas695** [NVIDIA/DeepLearningExamples#1292]
Related to **brats22/nnunet/pytorch** 

this is an inquiry for further explanation.

**Describe the bug**
this line of code : `  c1, c2, c3 = pred[0] > 0.4, pred[1] > 0.35, pred[2] > 0.375` in the `def to_lbl(pred):` function in the post-processing section of the brats22 notebook results in predicting that the entire area around the brain is enhancing tumor (label 4)  like in the picture below
[wrong prediction](https://drive.google.com/file/d/17jDaWPR1LFw2vIhRVS5otImz0HEpZAZZ/view?usp=share_link)
it is note worthy that i did the training on only the first 50 subjects of the brats dataset and with a patch size of  [80,80,80] and for 30 epochs to accommodate my hardware limitations
so can that be the cause the model is still predicting wrong ?
also something important to note that if i change the threshold to 0.5 instead of 0.4 for c1 this weird artifact doesn't happen like the picture below 
[correct prediction](https://drive.google.com/file/d/1E0DKHb_QqzkT7sajX2VEXJWQI09lyq7c/view?usp=share_link)
**To Reproduce**
follow the brats 22 notebook with the parameters I mentioned above for my training until the post-processing part 
**Expected behavior**
I expected that the right prediction will the one produced from the 0.4 threshold 

**Environment**
colab free environment (cuda 1.18 + torch 2.0 )

**Comment by michal2409** [comment]
The post-processing thresholds were tuned to our models, which were trained on the full BraTS dataset, 1000 epochs and patch [128, 128, 128], so the difference might come from using different training settings. Feel free to tune the thresholds so that your model gives you the best performance.

**Comment by abbas695** [comment]
thank you

**Comment by michal2409** [comment]
Dice is the mean dice value of D1, D2, D3 which are dice values for each label (1, 2, 3). Both are from the validation set.

**Comment by abbas695** [comment]
thank you for the clarification
